server/index.js

```diff
@@ -62,7 +62,7 @@ io.on('connection', (socket) => {
         console.log('getting top 10')
         getTop10().then((top10) => {
             // TODO add call to redis to initialize swiping room here
-            io.to(user.room).emit('sessionMembers', { room: user.room, users: getUsersInRoom(user.room), host: getHost(user.room), top10: top10});
+            io.to(user.room).emit('sessionMembers', { roomId: user.room, users: getUsersInRoom(user.room), host: getHost(user.room), top10: top10});
         }).catch((error) => {
             console.log('error getting list of top 10 movies from rapidapi')
```

Comment:
I think we should include the error in the console.log() method. Something like: `console.log('error getting list of top 10 movies from rapidapi', error)`

Reply:
sounds good